packages/fluentui/react-northstar/src/themes/teams/components/Card/cardStyles.ts

```diff
@@ -39,25 +42,32 @@ const cardStyles: ComponentSlotStylesPrepared<CardStylesProps, CardVariables> =
       ...(p.actionable && {
         cursor: 'pointer',
         ':focus-visible': {
+          backgroundColor: v.backgroundColorFocus,
           boxShadow: v.boxShadowFocus,
         },
         ':active': {
+          backgroundColor: v.backgroundColorPressed,
+          borderColor: v.borderColorPressed,
           boxShadow: v.boxShadowPressed,
         },
       }),
 
       ...(p.disabled && {
         cursor: 'not-allowed',
         color: v.colorDisabled,
+        backgroundColor: v.backgroundColorDisabled,
         borderColor: v.borderColorDisabled,
         boxShadow: v.boxShadowDisabled,
         ':hover': {
+          backgroundColor: v.backgroundColorDisabled,
```

Comment:
why were these 3 needed? is there an override between them and the one on line 58?

Reply:
Nope, good point, will remove them :)